### PR TITLE
add image to readme

### PR DESCRIPTION
**Pull request by salvadorstrambini**
## Motivation

It is a practice for making PR's. My goal is add one image to the Readme file.

## Summary of changes

Added one image to the readme file.
I titled the image "bannerimage".

## Checklist
- [ ] I have tested the changes locally.
- [ ] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

File: IMG_9706.PNG
Comment (AngieDutra) [inline IMG_9706.PNG:1]: Avoid unclear names for any type of file. Maybe "ChampionsBattle" is a good name for this image

Comment (AngieDutra) [inline IMG_9706.PNG:1]: This file should be inside of a "docs" folder

**Review by AngieDutra — CHANGES_REQUESTED** [review]
Apart from your PR, we have broken links in main. Maybe just delete those lines to have a tidier read me. We can add more images later. The following lines should be deleted:

```
<div>
  <div float="center">
    <img src="docs/src/images/Curse_of_Myrra_3D_Assets_Muflus.png" alt="Muflus 3D model" width=300px>  
    <img src="docs/src/images/Curse_of_Myrra_3D_Assets_Uma.jpeg" alt="Uma 3D model" width=300px> 
  </div>
  <div float="center">
    <img src="docs/src/images/Curse_of_Myrra_concept_art_Shinko.png" alt="Shinko hero concept art" width=300px>
    <img src="docs/src/images/Curse_of_Myrra_concept_art_Otobi_dog.png" alt="Concept art for a gang member dog in the planet of Otobi" width=300px>
  </div>
<div>
```

File: IMG_9706.PNG
Comment (AngieDutra) [reply IMG_9706.PNG:1]: Just an observation you should add the docs path if you create the folder requested. The readme addition would be:
`Champions Banner`

Comment (salvadorstrambini) [reply IMG_9706.PNG:1]: Ok thanks

Comment (salvadorstrambini) [reply IMG_9706.PNG:1]: Done. I think is alright now. Thank you

**Review by AngieDutra — APPROVED** [review]
Good one!

**Review by ncontinanza — APPROVED** [review]
Nice!

**Pull request merged**